src/Sylius/Behat/Element/Admin/CatalogPromotion/FormElement.php

```diff
@@ -51,6 +51,11 @@ public function checkChannel(string $channelName): void
         $this->getDocument()->checkField($channelName);
     }
 
+    public function checkExclusive(): void
+    {
+        $this->getElement('exclusive')->setValue(true);
+    }
```

Comment:
```suggestion
    public function setExclusiveness(bool $isExclusive): void
    {
        $this->getElement('exclusive')->setValue($isExclusive);
    }
```
it will make this function more reusable, WDYT?